docs/connectors/index.mdx

```diff
@@ -6,37 +6,35 @@ on a growing library of **[input](sources)** and **[output](sinks)** connectors.
 
 ## Basics
 
-A connector is a Feldera API object that describes an external data source or sink such as a database
-table or a Kafka topic. Connectors work together with other API objects, namely programs and pipelines,
-to build end-to-end streaming analytics pipelines.
-
-* A [program](/docs#programs) defines a set of SQL tables and views.
-* A [pipeline](/docs#pipelines) consists of a program with input and output connectors attached to it.
-* A connector must explicitly be attached to a table (for input)
-  or view (for output) of a pipeline.
-* A connector can be attached to the tables/views of one or more pipelines,
-  and can even be attached to multiple tables (for input) or views (for output)
-  within the same pipeline.
-* Connectors are managed through the API under the **/v0/connectors/...** endpoints.
-  They can be retrieved
-  [individually](https://www.feldera.com/api/fetch-a-connector-by-name) or
-  [as a list](https://www.feldera.com/api/fetch-connectors-optionally-filtered-by-name-or-id),
-  [created](https://www.feldera.com/api/create-a-new-connector),
-  [overwritten](https://www.feldera.com/api/create-or-replace-a-connector),
-  [modified](https://www.feldera.com/api/update-the-name-description-and-or-configuration-of-a-connector) and
-  [deleted](https://www.feldera.com/api/delete-an-existing-connector).
-* If changes are made to the connectors of a running pipeline, they will only be
-  applied when the pipeline is restarted.
+A connector is a JSON object that describes an external data source or sink such as a database
+table or a Kafka topic. They are specified as a list of JSON connector objects
+under the `'connectors'` attribute in the `WITH` clause of the table.
+
+Here is an example, where the `VENDOR` has one connector, configured to fetch data in a
+JSON format using HTTP and insert it:
+
+```sql
+create table VENDOR (
+    id bigint not null primary key,
+    name varchar,
+    address varchar
+) WITH ('connectors' = '[{
+    "transport": {
+        "name": "url_input", "config": {"path": "https://feldera-basics-tutorial.s3.amazonaws.com/vendor.json"}
+    },
+    "format": { "name": "json" }
+}]');
+```
```

Comment:
Likewise, a SQL view can have one or more sink connectors:

```sql
this needs an example, especially because the syntax for views is slightly different 
```

Reply:
what's the difference? the with can't be at the end?

Reply:
yep